src/app/backend/validation/validateloginstatus.go

```diff
@@ -32,13 +32,18 @@ type LoginStatus struct {
 }
 
 // ValidateLoginStatus returns information about user login status and if request was made over HTTPS.
-func ValidateLoginStatus(request *restful.Request) *LoginStatus {
+func ValidateLoginStatus(request *restful.Request, enableInsecureLogin bool) *LoginStatus {
 	authHeader := request.HeaderParameter("Authorization")
 	tokenHeader := request.HeaderParameter(client.JWETokenHeader)
 
+	httpsMode := request.Request.TLS != nil
+	if enableInsecureLogin {
+		httpsMode = true
+	}
```

Comment:
Looks a bit strange, because it could be written as `httpsMode = enableInsecureLogin`.